### PR TITLE
feat/DAT-2739:updates to the follow_up node

### PR DESCRIPTION
**Pull request by tlicht3**
https://jira.opensciencedatacloud.org/browse/DAT-2739

### CODE REVIEW

File: gdcdictionary/schemas/follow_up.yaml [hunk 1]
```diff
@@ -1729,12 +1900,270 @@ properties:
...
+    enum:
+      - Candidiasis
+      - Cytomegalovirus
+      - Coccidioidomycosis
+      - Cryptococcosis
+      - Encephalopathy
+      - Herpes Simplex Virus
+      - Histoplasmosis
+      - Isosporiasis
+      - Mycobacterium avium complex
```
Comment (wwysoc2) [inline gdcdictionary/schemas/follow_up.yaml]: Should 'complex' be capitalized?  I understand that "avium" and "tuberculosis" are probably species names right?   @tlicht3

File: gdcdictionary/schemas/follow_up.yaml [hunk 2]
```diff
@@ -1693,18 +1836,33 @@ properties:
...
+      - Sialadenitis
+      - Squamous Metaplasia
       - Steatosis
       - Tattoo
       - Thyroid Nodular Hyperplasia
       - Tobacco, NOS
       - Tobacco, Smokeless
       - Tobacco, Smoking
+      - Tubulointerstitial Disease
+      - Tumor-associated lymphoid proliferation
```
Comment (wwysoc2) [inline gdcdictionary/schemas/follow_up.yaml]: Should "associated", "lymphoid", "proliferation" be capitalized? @tlicht3 @ganjigajanan

File: gdcdictionary/schemas/follow_up.yaml [hunk 3]
```diff
@@ -1667,17 +1806,21 @@ properties:
       - Hepatitis B Infection
       - Hepatitis C Infection
       - Hepatitis, NOS
+      - Hereditary Prostate Cancer
       - High Grade Dysplasia
       - HIV
```
Comment (wwysoc2) [inline gdcdictionary/schemas/follow_up.yaml]: Should "High-grade Prostatic Intraepithelial Neoplasia (PIN)" be around here?

File: gdcdictionary/schemas/follow_up.yaml (hunk 1 above)
Comment (tlicht3) [reply gdcdictionary/schemas/follow_up.yaml]: @wwysoc2 yes we can update this.

**Pull request merged**